src/components/bodyImageThumbnail.tsx

```diff
@@ -0,0 +1,35 @@
+// ----- Imports ----- //
+
+import React, { FC, ReactNode } from 'react';
+import { css } from '@emotion/core';
+import Image, { Props as ImageProps } from 'components/image';
+import { Pillar } from 'pillar';
+
+// ----- Setup ----- //
+
+const sizes = `(min-width: 140px)`;
+
+
+// ----- Component ----- //
+
+interface Props {
+    image: Omit<ImageProps, 'sizes'>;
```

Comment:
What's that `Omit` type?

Reply:
Docs [here](https://www.typescriptlang.org/docs/handbook/utility-types.html#omittk). I think this is derived from the `Props` I created for the `Image` component.

I haven't yet decided whether it was a good idea or not. I'm a bit cautious about making use of utility types... it often feels like you're getting round the fact that if you'd designed your types another way you wouldn't have this problem...